### PR TITLE
[NMA-1259] Changing font for the new styles & fixing some UI issues

### PR DESCRIPTION
**Pull request by Syn-McJ**
As the first part of changing the font in the app, we want to move the new styles and features that use them from Montserrat to Inter.

## Issue being fixed or feature implemented
- Changed the font in the new styles
- Fixed/updated some UI that is out of date with Figma

## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [x] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

**Review by HashEngineering — APPROVED** [review]
Looks good to me.

File: wallet/res/layout/unable_to_connect_view.xml [hunk 1]
```diff
@@ -1,72 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<merge xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (HashEngineering) [inline wallet/res/layout/unable_to_connect_view.xml:2]: I guess with the new method of showing blockchain sync "Syncing 30%" on the home screen, we may no longer have a means of telling the user that the device is offline.

File: wallet/res/layout/verify_seed_success.xml [hunk 2]
```diff
@@ -1,48 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (HashEngineering) [inline wallet/res/layout/verify_seed_success.xml:2]: I guess we never used this.

**Review by HashEngineering — APPROVED** [review]
Looks good.

File: wallet/res/layout/unable_to_connect_view.xml (hunk 1 above)
Comment (Syn-McJ) [reply wallet/res/layout/unable_to_connect_view.xml:2]: I mean we still have this red panel on the home screen when the device is offline, no? This particular layout wasn't used anywhere.

**Pull request merged**